tools/internal/config/config.go

```diff
@@ -125,9 +125,15 @@ func (image *Image) setDefaults() error {
 	if len(parts) < 2 {
 		return fmt.Errorf("source image split into %d parts (>=2 parts expected)", len(parts))
 	}
-	repoName := parts[len(parts)-2]
-	imageName := parts[len(parts)-1]
-	image.defaultTargetImageName = "mirrored-" + repoName + "-" + imageName
+
+	if parts[0] == "dp.apps.rancher.io" {
```

Comment:
For posterity, perhaps adding a comment here mirroring the contents of the PR would be good, like
```go
// appco images have only one significant part in their reference because they take the form `dp.apps.rancher.io/containers/openjdk`
```